PROPOSALS.md

```diff
@@ -78,11 +78,29 @@ request feedback on feasibility before investing time and effort in the full pro
     * or just share patch files, in the same format as
       [`head.hackage`](https://gitlab.haskell.org/ghc/head.hackage) does.
 5. Implement the proposal.
-    * Note that you should follow [GHC's "Contributing a Patch" guidelines](https://gitlab.haskell.org/ghc/ghc/-/wikis/Contributing-a-Patch) when creating an MR (e.g. [appropriate commit messages](https://gitlab.haskell.org/ghc/ghc/-/wikis/Contributing-a-Patch#22-commit-messages)),
+    * Register at https://gitlab.haskell.org/ghc/ghc.
+    * If your account is not approved soon (this is a manual process to combat spam),
+      ask for approval in your CLC discussion thread, someone will take care of it.
+    * Fork GHC repository and implement your proposal in a non-`master` branch.
+      Do not work in `master`, it will cause issues with Marge Bot later.
+    * Follow [GHC's "Contributing a Patch" guidelines](https://gitlab.haskell.org/ghc/ghc/-/wikis/Contributing-a-Patch)
+      when creating an MR (e.g. [appropriate commit messages](https://gitlab.haskell.org/ghc/ghc/-/wikis/Contributing-a-Patch#22-commit-messages)).
     * Make sure to include a link to the CLC discussion into the commit message.
-    * Make sure to update `libraries/base/changelog.md`.
+    * Make sure to update `libraries/base/changelog.md`. Insert a new entry
+      at an arbitrary position instead of the last one to minimise merge conflicts.
```

Comment:
Who does it impact if this instruction is not followed? If it's the MR author (as opposed to other users of the `ghc` repo) then can we make it a suggestion instead? (I think that minimizing the number of absolutely required instruction steps will be helpful to new users.)

```suggestion
    * Make sure to update `libraries/base/changelog.md`. (You might like to insert your new entry
      at an arbitrary position instead of the last one to minimise merge conflicts.)
```

Reply:
It primarily impacts me ;) when proposals get stuck on a merge conflict and I have to nag people to resolve it. It happens surprisingly often and costs me a lot of operational costs.

Reply:
Fair enough